content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0. The latest release includes over a dozen updates and new features, and it is the first step toward Sourcegraph becoming a code intelligence platform. 
+tags: [blog, release]
+slug: "release/4.0"
+published: true
```

Comment:
@iskyOS IMO we should add authors: whoever primarily wrote the main part, and then the authors of each section. even if it's a lot of authors. if this is too complex to have a lot of authors, feel free to disregard.

Reply:
I like this idea @sqs! I don't know if our blog rendering will support that # of authors, but we can test it. I'd also like @lcolston08 to help figure out who those authors should be (and she was the main author).

My recommendation for now is to go forward with the blog as-is, and in the morning we can test the limits on # authors (and Lori can help us figure out who those authors are). We can also merge that as a new PR shortly after this goes live